pages/[id].js

```diff
@@ -0,0 +1,56 @@
+import { useRouter } from "next/router";
+import Header from "@/components/Header/Header";
+import prioColor from "@/utils/prioColor";
+import Link from "next/link";
+import BackButton from "@/components/BackButton/BackButton";
```

Comment:
As mentioned before, this actually should be a link and not a button. Therefore I suggest to rename the component accordingly.

Reply:
We wanted to create a reusable Button component that is on the same spot across all pages and also serves as the "Create Button" later on. The create mode will stay on the main page and will only be a dialogue window, which is why we cannot use a link for that. Should we still change it to a Link? In that case we would not be able to reuse the component